bugbug/bug_features.py

```diff
@@ -558,11 +558,10 @@ def get_time_to_assign(bug):
                 and change["removed"] in ("UNCONFIRMED", "NEW")
                 and change["added"] == "ASSIGNED"
             ):
-                return (
-                    dateutil.parser.parse(history["when"])
-                    - dateutil.parser.parse(bug["creation_time"])
-                ).total_seconds() / 86400
-
+                return utils.get_business_days_count(
+                    bug["creation_time"],
+                    history["when"],
+                )
```

Comment:
The `time_to_assign` feature uses this method; thus, we should not alter its behaviour to keep consistency with other time features.

I would suggest pulling up the logic that extracts the start and the end date in its own function and using it in a new function (e.g., `get_time_to_assign_busdays`) to count the number of business days. Also, we can use it in this existing function (i.e., `get_time_to_assign`) at the same time.

Reply:
> Also, we can use it in this existing function (i.e., get_time_to_assign) at the same time.
What do you mean by this? I've made a new function `get_time_to_assign_busdays` but I'm not understanding where to incorporate it in `get_time_to_assign`.

Reply:
I mean we should not change the behaviour of the `get_time_to_assign()` function since it is used somewhere else. Instead, creating a new function to do what we want.

Reply:
Okay, went ahead and put it as its own function, though nothing calls it at the moment. Should I add that somewhere?